_includes/domain-search.html

```diff
@@ -0,0 +1,100 @@
+{% comment %}
+TODO: Domain search description
+{% endcomment %}
+
+<script>
+    function checkDomainAvailability() {
+        // TODO: update baseUrl to stable when Cloudflare migrates their data 11/15
+        const baseUrl = "https://getgov-staging.app.cloud.gov/api/v1/available/"
+
+        var requested_domain = document.getElementById('domain-input').value
+
+        if (requested_domain === null || requested_domain.length === 0) {
+            render_result(``)
+        }
+
+        var searchEndpoint = new URL(requested_domain, baseUrl);
+
+        fetch(searchEndpoint).then(function(reply) {
+            return reply.json()
+        }).then(function(response) {
+            if (response.available) {
+                render_result(`
+                <div class="usa-alert usa-alert--success usa-alert--slim" role="alert">
```

Comment:
**Comment** I do wonder if there is a cleaner way to do this so the html isn't a string on the render, nested in code. But I wouldn't block on this

Reply:
Def something to look into! We were following the pattern `search.html` had just to start :)